crates/ruff_cli/resources/test/fixtures/cache_mutable/source.py

```diff
@@ -1,3 +0,0 @@
-a = 1
-
-__all__ = list(["a", "b"])
```

Comment:
What's the deal with this file? Is it just not used at all?

Reply:
The test cases create this file

https://github.com/astral-sh/ruff/blob/7f1456a2c9019f543fde71493aa476f132c84a57/crates/ruff_cli/src/cache.rs#L471